src/backend/distributed/commands/alter_table.c

```diff
@@ -1517,6 +1522,41 @@ CreateWorkerChangeSequenceDependencyCommand(char *sequenceSchemaName, char *sequ
 }
 
 
+/*
+ * GetUsingClauseWithAccessMethodIfExists generates and returns "USING ..." string
+ * if there's an access method set to the view with the given oid.
+ */
+static char *
+GetUsingClauseWithAccessMethodIfExists(Oid viewOid)
+{
+	StringInfo usingClause = makeStringInfo();
+	Relation relation = try_relation_open(viewOid, AccessShareLock);
+	if (relation == NULL)
+	{
+		ereport(ERROR, (errmsg("cannot complete operation "
+							   "because no such view exists")));
+	}
+
+	Oid accessMethodOid = relation->rd_rel->relam;
+	if (OidIsValid(accessMethodOid))
+	{
+		HeapTuple amTuple = SearchSysCache1(AMOID,
```

Comment:
I think you can use `get_am_name` function here.

Reply:
Will check that out